src/components/Skeleton/Skeleton.tsx

```diff
@@ -15,7 +15,7 @@ export const Skeleton = ({ className, ...props }: SkeletonProps) => {
       className={classNames(skeleton, className)}
       backgroundColor="default3"
       width="100%"
-      height={1}
+      height={4}
```

Comment:
question: Can you tell me why? I'm asking because I'm concerned that some views in Dashboard might be affected and don't look well after this change, for example discount rules

Reply:
Skeleton by default looks like this all over dashboard
![CleanShot 2024-07-23 at 10 03 00](https://github.com/user-attachments/assets/f7bf4aee-a062-48fe-84b7-12b399df9a32)
It makes more sense to change it here :) 

I checked discounts and they look good

Reply:
![CleanShot 2024-07-23 at 10 06 24](https://github.com/user-attachments/assets/632c28ae-4d32-4b91-b22d-2cd51b8d19e3)

Reply:
Please confirm this change with Slawek and @andrzejewsky